odoo/cli/i18n.py

```diff
@@ -0,0 +1,164 @@
+import csv
+import io
+import logging
+from contextlib import closing
+from itertools import product
+from pathlib import Path
+
+import odoo
+from odoo.cli.command import Command
+from odoo.modules import get_module_path
+from odoo.modules.registry import Registry
+from odoo.tools import config
+from odoo.tools.translate import load_language, trans_export
+
+_logger = logging.getLogger(__name__)
+
+
+class I18n(Command):
+    """ Import, export, setup languages and internationalization files """
+
+    def run(self, cmdargs):
+        subparsers = self.parser.add_subparsers(dest='subcommand', required=True, help='Subcommands help')
+
+        # Import parser
+        import_parser = subparsers.add_parser('import', help="Import i18n files")
+        import_parser.add_argument('-c', dest='config',
+            help='Use a specific configuration file')
+        import_parser.add_argument(
+            '--format', '-f', dest='format', default='po', choices=('po', 'tgz', 'csv'),
+            help=("Export format, default 'po'"))
+        import_parser.add_argument(
+            '--lang', '-l', dest='lang', nargs='*', default=['pot'], metavar='LANG,...',
+            help=("List of language ISO codes to be exported, default 'pot' for template"))
+        import_parser.add_argument('modules', nargs='*',
+            metavar='MODULE,...', help=("Comma-separated list of modules to be exported"))
+
+        # Export parser
+        export_parser = subparsers.add_parser('export', help="Export i18n files")
+        export_parser.add_argument('-c', dest='config',
+            help='Use a specific configuration file')
+        export_parser.add_argument('--stdout', dest='to_stdout', action='store_true', default=False,
+            help="Use standard output")
+        export_parser.add_argument(
+            '--format', '-f', dest='format', default='po', choices=('po', 'tgz', 'csv'),
+            help=("Export format, default 'po'"))
+        export_parser.add_argument(
+            '--lang', '-l', dest='lang', nargs='*', default=['pot'], metavar='LANG,...',
+            help=("List of language ISO codes to be exported, default 'pot' for template"))
+        export_parser.add_argument('modules', nargs='+',
+            metavar='MODULE', help=("Modules to be exported"))
+
+        # Loadlang parser
+        loadlang_parser = subparsers.add_parser('loadlang', help="Load languages")
+        loadlang_parser.add_argument('-c', dest='config',
+            help='Use a specific configuration file')
+        loadlang_parser.add_argument(
+            '--languages', nargs='*', required=True,
```

Comment:
nargs='*' and required=True are inconsistent, do you mean nargs='+'?
also my point about nargs vs action=append